Shift3/boilerplate-client-react
# Add 'cross-env' to dependencies

## Changes
1. Moved `cross-env` to a dependency.

## Purpose
This will allow `[yarn start]` to run correctly with no errors.

## Learning
Please visit the comments from Issue #233 .

## Testing
1. Pull in the changes to your local copy of this branch.
2. Run `[yarn start]` and make sure there are no errors.

## Screenshot
![Screen Shot 2021-07-06 at 10 37 15 AM](https://user-images.githubusercontent.com/59614789/124618862-2719a080-de46-11eb-90bc-2b8a30ec4ed2.png)

## References #233
